consensus/vbft/service.go

```diff
@@ -2080,6 +2093,18 @@ func (self *Server) makeProposal(blkNum uint32, forEmpty bool) error {
 	return self.broadcast(proposal)
 }
 
+func (self *Server) getBlockTimestamp(blkNum uint32) (uint32, error) {
```

Comment:
what server.getBlockTimestamp(blkNum) means?
server.getBlockTimestamp(0), will return timestamp of block 0?

This is not a general function, suggest pulling these back to contructProposalMsg.